sklearn/model_selection/_split.py

```diff
@@ -784,6 +790,8 @@ class TimeSeriesSplit(_BaseKFold):
     + n_samples % (n_splits + 1)`` in the ``i``th split,
     with a test set of size ``n_samples//(n_splits + 1)``,
     where ``n_samples`` is the number of samples.
+
+    ..versionadded:: v0.18
```

Comment:
```suggestion
    .. versionadded:: v0.18
```

Reply:
This should be above, in the summary section